<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDialog</class>
 <widget class="QDialog" name="PrefsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prefs - Amiga Cross Editor </string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_SavePrefs">
     <property name="text">
      <string>Save Prefs</string>
     </property>
     <property name="icon">
      <iconset resource="application.qrc">
       <normaloff>:/images/prefs.png</normaloff>:/images/prefs.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>297</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProject">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Author:</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_author</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_author">
         <property name="minimumSize">
          <size>
           <width>142</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>author_name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Email:</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_email</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_email">
         <property name="minimumSize">
          <size>
           <width>142</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>someone@somewhere.com</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Website:</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_website</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_website">
         <property name="minimumSize">
          <size>
           <width>142</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>http://www.somewhere.com</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Projects root:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>65</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGCC">
      <attribute name="title">
       <string>GCC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>GNU gcc:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_2">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>GNU g++:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_3">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_4">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>GNU make:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_28">
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_17">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>GNU strip:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_20">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_5">
           <property name="text">
            <string>-std=gnu89 -O2 -Wall -noixemul -lamiga</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>default opts:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVBCC">
      <attribute name="title">
       <string>VBCC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_20">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>vc: </string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_11">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>vasm: </string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_11">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_9">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_17">
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_10">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>config dir:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_12">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_19">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_14">
           <property name="text">
            <string>-v -O2 -size -lamiga</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>default opts:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEmulator">
      <attribute name="title">
       <string>Emulator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_27">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_22">
         <item row="0" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>UAE:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_16">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_13">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_21">
         <item row="0" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>OS 1.3 config:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_15">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_12">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_23">
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_14">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>OS 2.0 config:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_17">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_24">
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_15">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>OS 3.x config:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_18">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout" name="gridLayout_25">
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_getProjectRoot_16">
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>OS 4.x config:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_projects_root</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_projects_root_19">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout_26">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Default config:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBox">
           <property name="toolTip">
            <string>Select default AmigaOS to be started</string>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>AmigaOS 1.3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AmigaOS 2.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AmigaOS 3.x</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AmigaOS 4.x</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMisc">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_29">
       <item row="0" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>77</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>No nothing so far!</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>76</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
